Metronome/Metronome/ViewModel/MetronomeViewModel.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  MetronomeViewModel.swift
+//  Metronome
+//
+//  Created by Yuriy Egorov on 12/04/2023.
+//
+
+import AVKit
+import SwiftUI
+
+let url = Bundle.main.url(forResource: "tick_sound", withExtension: "wav")!
+
+class MetronomeViewModel : ObservableObject {
+    @Published var player = try! AVAudioPlayer(contentsOf: url)
```

Comment:
```suggestion
    private let player = try? AVAudioPlayer(contentsOf: url)
```
Это сделает AVAudioPlayer опциональным, но в таком случае лучше не крэшить создание view model, а показать ошибку в UI, либо просто кинуть assert (будет крашиться на дев сборке, чтобы ты как разработчик смог поймать эту ошибку, но на прод сборке просто звук не будет воспроизводится)